addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5420,7 +5420,17 @@ msgctxt "#13005"
 msgid "Shutdown"
 msgstr ""
 
-#empty strings from id 13006 to 13007
+#. Dialog to inform user about crash from executable add-on or prevented crash on library type (Header)
+#: xbmc/addons/binary/ExceptionHandling.cpp
+msgctxt "#13006"
+msgid "Fatal error"
+msgstr ""
+
+#. Dialog to inform user about crash from executable add-on or prevented crash on library type (Description)
+#: xbmc/addons/binary/ExceptionHandling.cpp
+msgctxt "#13007"
+msgid "The add-on '%s' has triggered a crash and is now switched off. Please contact the developer of him."
```

Comment:
here is a suggestion

The '%s' add-on has crashed and has been disabled. Please contact the developer for support.